src/main/java/com/google/api/generator/gapic/composer/grpc/GrpcServiceStubClassComposer.java

```diff
@@ -335,4 +447,120 @@ private LambdaExpr createRequestParamsExtractorClassInstance(Method method) {
         .setReturnExpr(returnExpr)
         .build();
   }
+
+  private void addRequestParamsForRoutingHeders(
+      Method method,
+      List<Statement> classStatements,
+      VariableExpr paramsVarExpr,
+      List<Expr> bodyExprs,
+      VariableExpr requestVarExpr) {
+    ImmutableList<RoutingHeader> routingHeaders = method.routingHeaders().routingHeadersList();
```

Comment:
I think `routingHeaders()` is nullable?

Reply:
The builder of RoutingHeaders will always create an empty map on initialization.